### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by yaira2**

### CODE REVIEW

**Comment by yaira2** [comment]
@chingucoding the CI issue is still happening, I wonder if someone messed up a translation string 🤔

**Comment by chingucoding** [comment]
Yeah, I think. The "Add new tab" button doesn't seem to have a UIA name anymore.

File: src/Files.App/Strings/en-GB/Resources.resw [hunk 1]
```diff
@@ -987,18 +987,15 @@
...
   <data name="HorizontalMultitaskingControlMoveTabToNewWindow.Text" xml:space="preserve">
     <value>Move tab to new window</value>
   </data>
-  <data name="HorizontalMultitaskingControlNewTab.Text" xml:space="preserve">
+  <data name="NewTab" xml:space="preserve">
     <value>New tab</value>
   </data>
-  <data name="NewTabCtrlT" xml:space="preserve">
-    <value>New tab (Ctrl+T)</value>
-  </data>
```
Comment (chingucoding) [inline src/Files.App/Strings/en-GB/Resources.resw:1001]: Removing this is causing the a11y issue, this key is needed for the new tabs button.

Comment (yaira2) [reply src/Files.App/Strings/en-GB/Resources.resw:1001]: I wonder why it wasn't caught when it was removed from the en-us file. Maybe it was falling back to this 🤔

Comment (chingucoding) [reply src/Files.App/Strings/en-GB/Resources.resw:1001]: I would definitely not be surprised if that was the case.

Comment (yaira2) [reply src/Files.App/Strings/en-GB/Resources.resw:1001]: @cinqmilleans can you add this back to the new tab button?

**Pull request merged**